src/modules/fetchingApi.js

```diff
@@ -0,0 +1,23 @@
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/lyyHMBWzgAOEuOOl3eGh/scores/';
+
+export const getScores = async () => {
+  const response = await fetch(apiUrl);
+  const jsonData = await response.json();
+  return jsonData.result;
+};
+
+export const addScore = async (user, score) => {
```

Comment:
- [ ] When I try to send a new score I got a network error and no indication in the front end of anything happened, kindly make sure to fix the issue or add instructions and validation if I need to do something else before I send the score to the board

![Screenshot_20230519_065336](https://github.com/JuanDiegoGuerra/Leaderboard/assets/25227022/d9d7b55c-e549-4adf-965a-c4afee933cf3)